v2/emailpassword/common-customizations/sessions/session-verification-in-api/verify-session.mdx

```diff
@@ -635,8 +635,9 @@ export function POST(request: NextRequest) {
     return withSession(request, async (session) => {
         let userId = session!.getUserId();
```

Comment:
when passing `{ sessionRequired: false }`, you need to check if session is undefined or not, and cannot do `session!`. See other code snippets for this block

Reply:
done.